mod-source-record-manager-server/src/main/resources/templates/db_scripts/schema.json

```diff
@@ -206,6 +206,16 @@
       "run": "after",
       "snippet": "CREATE INDEX IF NOT EXISTS journal_records_source_id_idx ON journal_records USING btree (source_id);",
       "fromModuleVersion": "mod-source-record-manager-3.4.0"
+    },
+    {
+      "run": "after",
```

Comment:
This is point for me... I need to re-check migration for srm more time after your changes will be merged to master.

Reply:
It would be good to re-check schema migration with this changes, thanks